<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Book Chapter &#8211; Jennifer Doe</title>
	<atom:link href="http://owwwlab.com/wp-faculty/pubtype/book-chapter/feed/" rel="self" type="application/rss+xml" />
	<link>http://owwwlab.com/wp-faculty</link>
	<description>Stanford University</description>
	<lastBuildDate>Thu, 05 Jul 2018 07:07:54 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	
	<item>
		<title>Floating search methods in feature selection</title>
		<link>http://owwwlab.com/wp-faculty/pubs/anomalously-increased-effective-thermal-conductivities-of-ethylene-glycol-based-nanofluids-containing-copper-nanoparticles-2/</link>
		<pubDate>Tue, 04 Nov 2014 04:39:52 +0000</pubDate>
		<dc:creator><![CDATA[Jennifer Doe]]></dc:creator>
		
		<guid isPermaLink="false">http://owwwlab.com/wp-faculty/publications/anomalously-increased-effective-thermal-conductivities-of-ethylene-glycol-based-nanofluids-containing-copper-nanoparticles-2/</guid>
		<description><![CDATA[Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure [&#8230;]]]></description>
				<content:encoded><![CDATA[<p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
]]></content:encoded>
			</item>
		<item>
		<title>An efficient crossover architecture for hardware parallel implementation of genetic algorithm</title>
		<link>http://owwwlab.com/wp-faculty/pubs/an-efficient-crossover-architecture-for-hardware-parallel-implementation-of-genetic-algorithm/</link>
		<pubDate>Sun, 09 Feb 2014 07:50:09 +0000</pubDate>
		<dc:creator><![CDATA[Jennifer Doe]]></dc:creator>
		
		<guid isPermaLink="false">http://localhost/wp-faculty/?post_type=publications&#038;p=392</guid>
		<description><![CDATA[Abstract A set of 134 SSR markers with high polymorphism covering the 17 linkage groups of pear were selected to [&#8230;]]]></description>
				<content:encoded><![CDATA[<div>
<h2 id="authorabs00101">Abstract</h2>
<p><a href="http://owwwlab.com/wp-faculty/wp-content/uploads/2014/02/1592223-e1391919876339.jpg"><img class="img-responsive" alt="1592223" src="http://owwwlab.com/wp-faculty/wp-content/uploads/2014/02/1592223-e1391919876339.jpg" width="1024" height="409" /></a></p>
<p id="spar0010">A set of 134 SSR markers with high polymorphism covering the 17 linkage groups of pear were selected to study the genetic variability and relationships of 99 <em>P. pyrifolia</em> cultivars. A total of 660 allelic variants were detected, with the number of observed alleles per locus varying from 3 to 9, with a mean of 4.93 alleles per locus. Among 81 SSR markers newly developed from the pear genome, the highest polymorphism was obtained by the motif (GT/CA), and the least polymorphism generated by (AC/TG). Clustering relationships of 99 <em>P. pyrifolia</em> cultivars generally reflected geographical classification, and further confirmed the close relationship between pears from Yangtze River Basin and Japan. The results also indicated that Sichuan province played an important role in pear resource distribution. Genetic structure analysis revealed the similar relationship as clustering, indicating the close relationships between Japanese and Chinese pear; and further proved more genetic exchange among cultivars from Japan and Zhejiang, and fewer genetic exchange among cultivars from Yunnan, Hubei and Sichuan. CoreS1, a core collection of <em>P. pyrifolia</em> newly constructed by the preferred sampling strategy, included 24 cultivars, retained 95.74% observed number of alleles (<em>Na</em>) of the 99 <em>P. pyrifolia</em> cultivars. Principal component analysis (PCA) further supported the representative genetic diversity of CoreS1 for <em>P. pyrifolia</em> cultivars from China, Japan, and Korea. The high quality and comprehensive evaluation of <em>P. pyrifolia</em> cultivars by the SSR markers covering the whole genome demonstrates their potential application in genetic diversity, genetic relationship, and core collection research on other germplasm resources of pear.</p>
</div>
<div id="SD_BA1P"></div>
<h2 id="kwd_1">Keywords</h2>
<ul id="kwd0005">
<li id="kw0005">Pear (<em>P. pyrifolia</em>);</li>
<li id="kw0010">SSR markers;</li>
<li id="kw0015">Genetic diversity;</li>
<li id="kw0020">Core collection</li>
</ul>
]]></content:encoded>
			</item>
		<item>
		<title>A fast and elitist multiobjective genetic algorithm</title>
		<link>http://owwwlab.com/wp-faculty/pubs/sciencedirect-paper-2002-2/</link>
		<pubDate>Sun, 09 Feb 2014 07:47:44 +0000</pubDate>
		<dc:creator><![CDATA[Jennifer Doe]]></dc:creator>
		
		<guid isPermaLink="false">http://owwwlab.com/wp-faculty/publications/sciencedirect-paper-2002-2/</guid>
		<description><![CDATA[Abstract Genetic parameters and genetic trends for staple length (SL), staple fineness (SF), fiber length (FL) and fiber diameter (FD) [&#8230;]]]></description>
				<content:encoded><![CDATA[<div>
<h2 id="authorabs00051">Abstract</h2>
<p id="spar0005">Genetic parameters and genetic trends for staple length (SL), staple fineness (SF), fiber length (FL) and fiber diameter (FD) were estimated using 7798 repeat records of 4583 Inner Mongolia cashmere goats at different ages measured from 2008–2011. These goats were descendants of 110 sires and 2139 dams, born between 2003 and 2010 at the Inner Mongolia Arbas cashmere goat stock farm. Analyses were carried out by average information restricted maximum likelihood, fitting four single-trait repeatability animal models with various combinations of individual and maternal effects. The best model for each fleece trait was chosen after testing for improvement of the log-likelihood values. Genetic parameters were then estimated under the most appropriate model. Genetic trends were determined by regressing yearly estimated mean breeding values on year of birth. All the fleece traits were influenced by direct addictive genetic and individual permanent environment effects. Estimates of direct heritability for SL, SF, FL and FD were 0.30, 0.27, 0.18 and 0.32, respectively. Genetic correlations among the fleece traits (SL–SF and FL–SF) were negative and high (−0.50 and −0.48). For the other fleece traits (SL–FL and SF–FD), the genetic correlation was moderate and positive (0.26 and 0.31). The phenotypic correlation among all fleece traits was low, ranging from −0.22 to 0.23. Compared with other traits, fleece traits were not objective traits in previous breeding programs; they presented irregular changes across year of birth. Estimates of genetic parameters and analyses of genetic trends would be helpful for designing a breeding program for the genetic improvement of fleece traits in Inner Mongolia cashmere goats.</p>
</div>
<div id="SD_BA1P"></div>
<h2 id="kwd_1">Keywords</h2>
<ul id="kwd0005">
<li id="kw0005">Heritability;</li>
<li id="kw0010">Genetic correlation;</li>
<li id="kw0015">Average breeding value;</li>
<li id="kw0020">Fleece traits</li>
</ul>
]]></content:encoded>
			</item>
		<item>
		<title>New paper</title>
		<link>http://owwwlab.com/wp-faculty/pubs/new-paper/</link>
		<pubDate>Sun, 09 Feb 2014 07:46:09 +0000</pubDate>
		<dc:creator><![CDATA[Jennifer Doe]]></dc:creator>
		
		<guid isPermaLink="false">http://localhost/wp-faculty/?post_type=publications&#038;p=390</guid>
		<description><![CDATA[Abstract We introduce new quantitative characteristics of the population using an analogy to the system of multi-spin molecules: the disease [&#8230;]]]></description>
				<content:encoded><![CDATA[<h3>Abstract</h3>
<div>
<p id="sp0075">We introduce new quantitative characteristics of the population using an analogy to the system of multi-spin molecules: the disease fields, which may depend on interactions, and the susceptibility to disease as derivative of genetic vector’s (GV’s) frequency of cases with respect to these fields. The genetic vector’s approach (GVA) is applied to statistical analysis of the interaction of two SNP haplotype of HTR2A and shared epitope (SE) alleles in relation to development of rheumatoid arthritis (RA). The analysis is performed for two independent cohorts, EIRA and NARAC, and based on the evaluation of double- and triple genotype–genotype versus SE alleles correlations. The Gibbs-like parametrization of GV frequencies makes analysis transparent and easy interpretable. We find that the main contribution into association to RA comes from GVs containing double SE. GVA may resolve an opposite role in risk/protection from different pairs of genetic variations and reveal an association to RA whereas the univariate analysis does not show significant association.</p>
<p>&nbsp;</p>
</div>
<div id="SD_BA1P"></div>
]]></content:encoded>
			</item>
		<item>
		<title>New paper Copy</title>
		<link>http://owwwlab.com/wp-faculty/pubs/new-paper-2/</link>
		<pubDate>Sun, 09 Feb 2014 07:46:09 +0000</pubDate>
		<dc:creator><![CDATA[Jennifer Doe]]></dc:creator>
		
		<guid isPermaLink="false">http://owwwlab.com/wp-faculty/publications/new-paper-2/</guid>
		<description><![CDATA[Abstract We introduce new quantitative characteristics of the population using an analogy to the system of multi-spin molecules: the disease [&#8230;]]]></description>
				<content:encoded><![CDATA[<h3>Abstract</h3>
<div>
<p id="sp0075">We introduce new quantitative characteristics of the population using an analogy to the system of multi-spin molecules: the disease fields, which may depend on interactions, and the susceptibility to disease as derivative of genetic vector’s (GV’s) frequency of cases with respect to these fields. The genetic vector’s approach (GVA) is applied to statistical analysis of the interaction of two SNP haplotype of HTR2A and shared epitope (SE) alleles in relation to development of rheumatoid arthritis (RA). The analysis is performed for two independent cohorts, EIRA and NARAC, and based on the evaluation of double- and triple genotype–genotype versus SE alleles correlations. The Gibbs-like parametrization of GV frequencies makes analysis transparent and easy interpretable. We find that the main contribution into association to RA comes from GVs containing double SE. GVA may resolve an opposite role in risk/protection from different pairs of genetic variations and reveal an association to RA whereas the univariate analysis does not show significant association.</p>
<p>&nbsp;</p>
</div>
<div id="SD_BA1P"></div>
]]></content:encoded>
			</item>
		<item>
		<title>A comparative study on feature selection</title>
		<link>http://owwwlab.com/wp-faculty/pubs/cultivating-admiration-in-brands-2/</link>
		<pubDate>Mon, 27 Jan 2014 16:42:07 +0000</pubDate>
		<dc:creator><![CDATA[Jennifer Doe]]></dc:creator>
		
		<guid isPermaLink="false">http://owwwlab.com/wp-faculty/publications/cultivating-admiration-in-brands-2/</guid>
		<description><![CDATA[Abstract Proven strategies for harnessing the power of social media to drive social change Many books teach the mechanics of [&#8230;]]]></description>
				<content:encoded><![CDATA[<h2>Abstract</h2>
<h4>Proven strategies for harnessing the power of social media to drive social change</h4>
<p>Many books teach the mechanics of using Facebook, Twitter, and YouTube to compete in business. But no book addresses how to harness the incredible power of social media to make a difference. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
<ul>
<li>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</li>
<li>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</li>
<li>.sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</li>
<li>Onsectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</li>
</ul>
]]></content:encoded>
			</item>
	</channel>
</rss>
